# Titanic competition with TensorFlow Decision Forests

This notebook will take you through the steps needed to train a baseline Gradient Boosted Trees Model using TensorFlow Decision Forests and creating a submission on the Titanic competition. 

This notebook shows:

1. How to do some basic pre-processing. For example, the passenger names will be tokenized, and ticket names will be splitted in parts.
1. How to train a Gradient Boosted Trees (GBT) with default parameters
1. How to train a GBT with improved default parameters
1. How to tune the parameters of a GBTs
1. How to train and ensemble many GBTs

# Imports dependencies

In [1]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
import tensorflow_decision_forests as tfdf

print(f"Found TF-DF {tfdf.__version__}")

Found TF-DF 1.8.1


# Load dataset

In [2]:
train_df = pd.read_csv("train.csv")
serving_df = pd.read_csv("test.csv")

train_df.head(10)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

# Prepare dataset

We will apply the following transformations on the dataset.

1. Tokenize the names. For example, "Braund, Mr. Owen Harris" will become ["Braund", "Mr.", "Owen", "Harris"].
2. Extract any prefix in the ticket. For example ticket "STON/O2. 3101282" will become "STON/O2." and 3101282.

In [3]:
def preprocess(df):
    df = df.copy()
    
    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])
    
    def ticket_number(x):
        return x.split(" ")[-1]
        
    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])
    
    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)                     
    return df
    
preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

Let's keep the list of the input features of the model. Notably, we don't want to train our model on the "PassengerId" and "Ticket" features.

In [4]:
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")
#input_features.remove("Ticket_number")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


# Convert Pandas dataset to TensorFlow Dataset

In [5]:
def tokenize_names(features, labels=None):
    """Divite the names into tokens. TF-DF can consume text tokens natively."""
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

2024-02-11 19:17:41.653517: I metal_plugin/src/device/metal_device.cc:1154] Metal device set to: Apple M2
2024-02-11 19:17:41.653543: I metal_plugin/src/device/metal_device.cc:296] systemMemory: 16.00 GB
2024-02-11 19:17:41.653549: I metal_plugin/src/device/metal_device.cc:313] maxCacheSize: 5.33 GB
2024-02-11 19:17:41.653620: I tensorflow/core/common_runtime/pluggable_device/pluggable_device_factory.cc:306] Could not identify NUMA node of platform GPU ID 0, defaulting to 0. Your kernel may not have been built with NUMA support.
2024-02-11 19:17:41.653908: I tensorflow/core/common_runtime/pluggable_device/pluggable_device_factory.cc:272] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 0 MB memory) -> physical PluggableDevice (device: 0, name: METAL, pci bus id: <undefined>)


# Train model with default parameters

### Train model

First, we are training a GradientBoostedTreesModel model with the default parameters.

In [6]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    random_seed=1234,
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-02-11 19:17:41.8359 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:17:41.8380 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:17:41.8380 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
2024-02-11 19:17:43.297069: I tensorflow/core/grappler/optimizers/custom_graph_optimizer_registry.cc:117] Plugin optimizer for device_type GPU is enabled.
[INFO 24-02-11 19:17:43.6519 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpw1_5aa4t/model/ with prefix 2bd535399a7d40b2
[INFO 24-02-11 19:17:43.6568 IST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-11 19:17:43.6568 IST kernel.cc:1061] Use fast generic engine


Accuracy: 0.804347813129425 Loss:0.8922085165977478


# Train model with improved default parameters

Now you'll use some specific parameters when creating the GBT model

In [7]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    
    #num_trees=2000,
    
    # Only for GBT.
    # A bit slower, but great to understand the model.
    # compute_permutation_variable_importance=True,
    
    # Change the default hyper-parameters
    # hyperparameter_template="benchmark_rank1@v1",
    
    #num_trees=1000,
    #tuner=tuner
    
    min_examples=1,
    categorical_algorithm="RANDOM",
    #max_depth=4,
    shrinkage=0.05,
    #num_candidate_attributes_ratio=0.2,
    split_axis="SPARSE_OBLIQUE",
    sparse_oblique_normalization="MIN_MAX",
    sparse_oblique_num_projections_exponent=2.0,
    num_trees=2000,
    #validation_ratio=0.0,
    random_seed=1234,
    
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-02-11 19:17:44.2284 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:17:44.2284 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:17:44.2284 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


Accuracy: 0.77173912525177 Loss:1.0136744976043701


[INFO 24-02-11 19:17:44.6237 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmphz_2j4qt/model/ with prefix d89f36abfa6a4a6a
[INFO 24-02-11 19:17:44.6284 IST decision_forest.cc:660] Model loaded with 54 root(s), 2824 node(s), and 10 input feature(s).
[INFO 24-02-11 19:17:44.6284 IST kernel.cc:1061] Use fast generic engine


Let's look at the model and you can also notice the information about variable importance that the model figured out

In [8]:
model.summary()

Model: "gradient_boosted_trees_model_1"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
Total params: 1 (1.00 Byte)
Trainable params: 0 (0.00 Byte)
Non-trainable params: 1 (1.00 Byte)
_________________________________________________________________
Type: "GRADIENT_BOOSTED_TREES"
Task: CLASSIFICATION
Label: "__LABEL"

Input Features (11):
	Age
	Cabin
	Embarked
	Fare
	Name
	Parch
	Pclass
	Sex
	SibSp
	Ticket_item
	Ticket_number

No weights

Variable Importance: INV_MEAN_MIN_DEPTH:
    1.           "Sex"  0.431313 ################
    2.           "Age"  0.373986 ############
    3.          "Fare"  0.250774 ####
    4.          "Name"  0.225272 ###
    5.   "Ticket_item"  0.182111 
    6.      "Embarked"  0.181387 
    7. "Ticket_number"  0.180897 
    8.        "Pclass"  0.178279 
    9.         "Parch"  0.175344 
   10.         "SibSp"  0.172167 

Variable Importance: NUM_AS_ROOT:
    1.  "Sex" 39.00000

# Make predictions

In [9]:
def prediction_to_kaggle_format(model, threshold=0.5):
    proba_survive = model.predict(serving_ds, verbose=0)[:,0]
    return pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (proba_survive >= threshold).astype(int)
    })

def make_submission(kaggle_predictions):
    path="submission.csv"
    kaggle_predictions.to_csv(path, index=False)
    print(f"Submission exported to {path}")
    
kaggle_predictions = prediction_to_kaggle_format(model)
make_submission(kaggle_predictions)

Submission exported to submission.csv


# Training a model with hyperparameter tunning

Hyper-parameter tuning is enabled by specifying the tuner constructor argument of the model. The tuner object contains all the configuration of the tuner (search space, optimizer, trial and objective).


In [10]:
tuner = tfdf.tuner.RandomSearch(num_trials=1000)
tuner.choice("min_examples", [2, 5, 7, 10])
tuner.choice("categorical_algorithm", ["CART", "RANDOM"])

local_search_space = tuner.choice("growing_strategy", ["LOCAL"])
local_search_space.choice("max_depth", [3, 4, 5, 6, 8])

global_search_space = tuner.choice("growing_strategy", ["BEST_FIRST_GLOBAL"], merge=True)
global_search_space.choice("max_num_nodes", [16, 32, 64, 128, 256])

#tuner.choice("use_hessian_gain", [True, False])
tuner.choice("shrinkage", [0.02, 0.05, 0.10, 0.15])
tuner.choice("num_candidate_attributes_ratio", [0.2, 0.5, 0.9, 1.0])


tuner.choice("split_axis", ["AXIS_ALIGNED"])
oblique_space = tuner.choice("split_axis", ["SPARSE_OBLIQUE"], merge=True)
oblique_space.choice("sparse_oblique_normalization",
                     ["NONE", "STANDARD_DEVIATION", "MIN_MAX"])
oblique_space.choice("sparse_oblique_weights", ["BINARY", "CONTINUOUS"])
oblique_space.choice("sparse_oblique_num_projections_exponent", [1.0, 1.5])

# Tune the model. Notice the `tuner=tuner`.
tuned_model = tfdf.keras.GradientBoostedTreesModel(tuner=tuner)
tuned_model.fit(train_ds, verbose=0)

tuned_self_evaluation = tuned_model.make_inspector().evaluation()
print(f"Accuracy: {tuned_self_evaluation.accuracy} Loss:{tuned_self_evaluation.loss}")

Use /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpum5ukzor as temporary training directory


[WARNING 24-02-11 19:17:44.8618 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:17:44.8618 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:17:44.8618 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


Accuracy: 0.8904109597206116 Loss:0.6763895750045776


[INFO 24-02-11 19:18:11.7664 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpum5ukzor/model/ with prefix b3825a0a20794f0f
[INFO 24-02-11 19:18:11.7747 IST decision_forest.cc:660] Model loaded with 26 root(s), 3162 node(s), and 12 input feature(s).
[INFO 24-02-11 19:18:11.7747 IST abstract_model.cc:1344] Engine "GradientBoostedTreesGeneric" built
[INFO 24-02-11 19:18:11.7747 IST kernel.cc:1061] Use fast generic engine


In the last line in the cell above, you can see the accuracy is higher than previously with default parameters and parameters set by hand.

This is the main idea behing hyperparameter tuning.

For more information you can follow this tutorial: [Automated hyper-parameter tuning](https://www.tensorflow.org/decision_forests/tutorials/automatic_tuning_colab)

# Making an ensemble

Here you'll create 100 models with different seeds and combine their results

This approach removes a little bit the random aspects related to creating ML models

In the GBT creation is used the `honest` parameter. It will use different training examples to infer the structure and the leaf values. This regularization technique trades examples for bias estimates.

In [11]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    # Possible models: GradientBoostedTreesModel or RandomForestModel
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0, # Very few logs
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True, # Only use the features in "features"

        #min_examples=1,
        #categorical_algorithm="RANDOM",
        ##max_depth=4,
        #shrinkage=0.05,
        ##num_candidate_attributes_ratio=0.2,
        #split_axis="SPARSE_OBLIQUE",
        #sparse_oblique_normalization="MIN_MAX",
        #sparse_oblique_num_projections_exponent=2.0,
        #num_trees=2000,
        ##validation_ratio=0.0,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)
    
    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

make_submission(kaggle_predictions)

[WARNING 24-02-11 19:18:11.8935 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:11.8936 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:11.8936 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:0


[INFO 24-02-11 19:18:12.0896 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmprt2vuoun/model/ with prefix 3cd94036b7ba424f
[INFO 24-02-11 19:18:12.0912 IST kernel.cc:1061] Use fast generic engine


[WARNING 24-02-11 19:18:12.2851 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:12.2851 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:12.2851 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:1


[INFO 24-02-11 19:18:12.5481 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp1vv5f2_8/model/ with prefix d87a2420c2e540b0
[INFO 24-02-11 19:18:12.5518 IST kernel.cc:1061] Use fast generic engine


[WARNING 24-02-11 19:18:12.7287 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:12.7287 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:12.7287 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:2


[INFO 24-02-11 19:18:13.0161 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp11nc11sx/model/ with prefix 7c33f55fde6b49d9
[INFO 24-02-11 19:18:13.0202 IST kernel.cc:1061] Use fast generic engine


[WARNING 24-02-11 19:18:13.1969 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:13.1970 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:13.1970 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:3


[INFO 24-02-11 19:18:13.4298 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp3ofa7bc2/model/ with prefix 2b8b674ebe734dea
[INFO 24-02-11 19:18:13.4326 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:13.6085 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:13.6085 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:13.6085 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:4


[INFO 24-02-11 19:18:13.8101 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpmy38ube0/model/ with prefix 6bfb9d7810c54b82
[INFO 24-02-11 19:18:13.8117 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:13.9858 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:13.9858 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:13.9858 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:5


[INFO 24-02-11 19:18:14.1959 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpj2mth9ln/model/ with prefix 72d532c278554d56
[INFO 24-02-11 19:18:14.1979 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:14.3706 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:14.3706 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:14.3706 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:6


[INFO 24-02-11 19:18:14.6180 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp47dk8lzq/model/ with prefix f2ae3bf7262d4631
[INFO 24-02-11 19:18:14.6212 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:14.7890 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:14.7890 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:14.7890 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:7


[INFO 24-02-11 19:18:15.0070 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpjqog2_rw/model/ with prefix 11e86d9b6e84411f
[INFO 24-02-11 19:18:15.0093 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:15.1834 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:15.1835 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:15.1835 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:8


[INFO 24-02-11 19:18:15.4650 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpli1xsj_m/model/ with prefix 560f6d7db6f54252
[INFO 24-02-11 19:18:15.4691 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:15.6388 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:15.6388 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:15.6389 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:9


[INFO 24-02-11 19:18:15.9216 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp49nuayoy/model/ with prefix b4f488df801d4b49
[INFO 24-02-11 19:18:15.9262 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:16.1003 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:16.1003 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:16.1003 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:10


[INFO 24-02-11 19:18:16.3197 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp0dschfxw/model/ with prefix 60be7883f1ab43c2
[INFO 24-02-11 19:18:16.3223 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:16.4952 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:16.4952 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:16.4952 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:11


[INFO 24-02-11 19:18:16.7981 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmprkuaghra/model/ with prefix 4a0bb1191c2444b1
[INFO 24-02-11 19:18:16.8031 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:16.9891 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:16.9891 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:16.9891 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:12


[INFO 24-02-11 19:18:17.2326 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpxrwyupy1/model/ with prefix c022bfa8856943e2
[INFO 24-02-11 19:18:17.2357 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:17.4138 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:17.4138 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:17.4138 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:13


[INFO 24-02-11 19:18:17.8389 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmplmvwwppd/model/ with prefix c5e70bc46b524dea
[INFO 24-02-11 19:18:17.8483 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:18.0214 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:18.0215 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:18.0215 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:14


[INFO 24-02-11 19:18:18.2705 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpe3ubdv_z/model/ with prefix d53a39c0414347d0
[INFO 24-02-11 19:18:18.2733 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:18.4535 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:18.4535 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:18.4535 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:15


[INFO 24-02-11 19:18:18.6578 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpz6xj9nqh/model/ with prefix b94aaa0c75404286
[INFO 24-02-11 19:18:18.6595 IST kernel.cc:1061] Use fast generic engine


i:16


[WARNING 24-02-11 19:18:18.9906 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:18.9906 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:18.9906 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-11 19:18:19.2463 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp0evtkpri/model/ with prefix a7e26f25a5944764
[INFO 24-02-11 19:18:19.2499 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:19.4359 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:19.4359 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:19.4359 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:17


[INFO 24-02-11 19:18:19.6546 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpfy4b9mwf/model/ with prefix 4136c0b65d06417d
[INFO 24-02-11 19:18:19.6569 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:19.8404 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:19.8404 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:19.8404 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:18


[INFO 24-02-11 19:18:20.1118 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp88xw1m12/model/ with prefix 06834a0cfa7d465b
[INFO 24-02-11 19:18:20.1160 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:20.2909 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:20.2909 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:20.2909 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:19


[INFO 24-02-11 19:18:20.5467 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpl26bajx6/model/ with prefix c3e13c5cbabf4ffb
[INFO 24-02-11 19:18:20.5504 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:20.7219 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:20.7219 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:20.7219 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:20


[INFO 24-02-11 19:18:20.9408 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpjsmi8yiw/model/ with prefix 93d8e3b3b3444ef6
[INFO 24-02-11 19:18:20.9431 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:21.1238 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:21.1238 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:21.1238 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:21


[INFO 24-02-11 19:18:21.3373 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp1w4o1hb5/model/ with prefix b292140d90aa4237
[INFO 24-02-11 19:18:21.3396 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:21.5130 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:21.5131 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:21.5131 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:22


[INFO 24-02-11 19:18:21.7604 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpwzmwebjk/model/ with prefix 180f206195f746da
[INFO 24-02-11 19:18:21.7636 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:21.9346 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:21.9346 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:21.9346 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:23


[INFO 24-02-11 19:18:22.1523 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp0x7kxc5s/model/ with prefix 46da3dfa62054532
[INFO 24-02-11 19:18:22.1549 IST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-11 19:18:22.1549 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:22.3524 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:22.3524 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:22.3524 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:24


[INFO 24-02-11 19:18:22.5745 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpg_kq5pcm/model/ with prefix 38d66a77d8b444f3
[INFO 24-02-11 19:18:22.5768 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:22.7604 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:22.7604 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:22.7604 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:25


[INFO 24-02-11 19:18:22.9992 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmprxidxmuq/model/ with prefix fffa0e1bf11642e3
[INFO 24-02-11 19:18:23.0022 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:23.1755 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:23.1755 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:23.1755 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:26


[INFO 24-02-11 19:18:23.3879 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpejhzj1l6/model/ with prefix 1c5abb0ad025418b
[INFO 24-02-11 19:18:23.3898 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:23.5721 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:23.5721 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:23.5721 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:27


[INFO 24-02-11 19:18:23.8452 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpddyfx0rm/model/ with prefix 430a6f50c14e4d9d
[INFO 24-02-11 19:18:23.8493 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:24.0436 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:24.0436 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:24.0436 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:28


[INFO 24-02-11 19:18:24.2499 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmph2mmnk0h/model/ with prefix e7243fac7ee74ad3
[INFO 24-02-11 19:18:24.2511 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:24.4309 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:24.4309 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:24.4309 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:29


[INFO 24-02-11 19:18:24.6411 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp09n5_td1/model/ with prefix 707a366d7bf44bd2
[INFO 24-02-11 19:18:24.6431 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:24.8387 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:24.8387 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:24.8387 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:30


[INFO 24-02-11 19:18:25.1391 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmph2qqcpa2/model/ with prefix 76c864d9449e439d
[INFO 24-02-11 19:18:25.1427 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:25.3327 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:25.3328 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:25.3328 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:31


[INFO 24-02-11 19:18:25.6616 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmppqn_7khz/model/ with prefix cfa7771dab8e48dd
[INFO 24-02-11 19:18:25.6670 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:25.8402 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:25.8402 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:25.8403 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:32


[INFO 24-02-11 19:18:26.0622 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpo6ydx3tl/model/ with prefix 54d1e9c802244918
[INFO 24-02-11 19:18:26.0648 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:26.2375 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:26.2375 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:26.2375 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:33


[INFO 24-02-11 19:18:26.4844 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpbri5l_e5/model/ with prefix 9150dc84d0ae4993
[INFO 24-02-11 19:18:26.4873 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:26.6618 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:26.6618 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:26.6618 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:34


[INFO 24-02-11 19:18:26.8836 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmphnn2zzor/model/ with prefix 9b939025d75747c6
[INFO 24-02-11 19:18:26.8861 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:27.0593 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:27.0593 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:27.0593 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:35


[INFO 24-02-11 19:18:27.3106 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpb40_eo_f/model/ with prefix 24da7d4680e24a2d
[INFO 24-02-11 19:18:27.3140 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:27.4925 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:27.4925 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:27.4925 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:36


[INFO 24-02-11 19:18:27.7159 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp1k1zlno5/model/ with prefix 96fed086da304f4c
[INFO 24-02-11 19:18:27.7180 IST kernel.cc:1061] Use fast generic engine


i:37


[WARNING 24-02-11 19:18:28.1034 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:28.1035 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:28.1035 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-11 19:18:28.3251 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp9zuyru4c/model/ with prefix fb3ed6d4f14c46a6
[INFO 24-02-11 19:18:28.3273 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:28.5153 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:28.5153 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:28.5154 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:38


[INFO 24-02-11 19:18:28.7654 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp7vre54e5/model/ with prefix 438877c1e6f74fff
[INFO 24-02-11 19:18:28.7686 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:28.9432 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:28.9432 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:28.9432 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:39


[INFO 24-02-11 19:18:29.2169 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpoya3zkzd/model/ with prefix 0787bc032bb64db8
[INFO 24-02-11 19:18:29.2207 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:29.3998 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:29.3999 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:29.3999 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:40


[INFO 24-02-11 19:18:29.6341 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpamsrih_8/model/ with prefix e3e4a108632849f8
[INFO 24-02-11 19:18:29.6365 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:29.8114 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:29.8114 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:29.8114 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:41


[INFO 24-02-11 19:18:30.1045 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp4_cq8o3t/model/ with prefix 7062fa5fdccc4dcb
[INFO 24-02-11 19:18:30.1082 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:30.2886 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:30.2886 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:30.2886 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:42


[INFO 24-02-11 19:18:30.5245 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp0dxsfpmm/model/ with prefix 007fb45997c14ce8
[INFO 24-02-11 19:18:30.5269 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:30.7040 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:30.7040 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:30.7040 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:43


[INFO 24-02-11 19:18:30.9637 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpn8fneduk/model/ with prefix 9295dafcb6e44887
[INFO 24-02-11 19:18:30.9671 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:31.1437 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:31.1437 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:31.1437 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:44


[INFO 24-02-11 19:18:31.4067 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpqwitofr4/model/ with prefix eea0933662344593
[INFO 24-02-11 19:18:31.4092 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:31.5913 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:31.5913 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:31.5913 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:45


[INFO 24-02-11 19:18:31.7867 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp0guhxgfx/model/ with prefix 8cbd7e43418b40ed
[INFO 24-02-11 19:18:31.7880 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:31.9697 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:31.9697 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:31.9697 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:46


[INFO 24-02-11 19:18:32.2658 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpdrglgich/model/ with prefix 9de4016fb6f14758
[INFO 24-02-11 19:18:32.2690 IST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-11 19:18:32.2690 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:32.4483 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:32.4483 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:32.4483 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:47


[INFO 24-02-11 19:18:32.6848 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpyvf3oxxf/model/ with prefix ce128e55101145ea
[INFO 24-02-11 19:18:32.6871 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:32.8658 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:32.8658 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:32.8658 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:48


[INFO 24-02-11 19:18:33.0807 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpm7oscu8y/model/ with prefix f178468195214b8c
[INFO 24-02-11 19:18:33.0826 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:33.2658 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:33.2658 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:33.2658 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:49


[INFO 24-02-11 19:18:33.5291 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpkqdjw80k/model/ with prefix cd88f03110d1452c
[INFO 24-02-11 19:18:33.5324 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:33.7198 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:33.7198 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:33.7198 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:50


[INFO 24-02-11 19:18:34.0221 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpfv_zsqen/model/ with prefix df81442a02714b94
[INFO 24-02-11 19:18:34.0265 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:34.2054 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:34.2054 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:34.2054 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:51


[INFO 24-02-11 19:18:34.5348 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpp8bxktgi/model/ with prefix 8ed30a3599784d7b
[INFO 24-02-11 19:18:34.5407 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:34.7239 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:34.7239 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:34.7240 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:52


[INFO 24-02-11 19:18:34.9592 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpe27gpil1/model/ with prefix 16f558e22b69429e
[INFO 24-02-11 19:18:34.9620 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:35.1428 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:35.1428 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:35.1428 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:53


[INFO 24-02-11 19:18:35.3754 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpdzhr3sow/model/ with prefix 1b4f8b6c6fff49bb
[INFO 24-02-11 19:18:35.3777 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:35.5564 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:35.5564 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:35.5564 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:54


[INFO 24-02-11 19:18:35.8099 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpc1suji6t/model/ with prefix 49ca1f5baa4346a4
[INFO 24-02-11 19:18:35.8130 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:35.9926 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:35.9926 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:35.9926 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:55


[INFO 24-02-11 19:18:36.2502 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp5dc7srg0/model/ with prefix e48709fd9ea94dfd
[INFO 24-02-11 19:18:36.2535 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:36.4364 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:36.4364 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:36.4364 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:56


[INFO 24-02-11 19:18:36.6720 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmply96qgo1/model/ with prefix 23ddcffeecde420e
[INFO 24-02-11 19:18:36.6746 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:36.8605 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:36.8605 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:36.8605 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:57


[INFO 24-02-11 19:18:37.0707 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpy8yrzgvc/model/ with prefix 1b12328abba042d5
[INFO 24-02-11 19:18:37.0722 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:37.2523 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:37.2523 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:37.2523 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:58


[INFO 24-02-11 19:18:37.5692 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpjs9xo5by/model/ with prefix 2c135090618c4e63
[INFO 24-02-11 19:18:37.5745 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:37.7565 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:37.7565 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:37.7565 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:59


[INFO 24-02-11 19:18:38.0701 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpj383k2_2/model/ with prefix 2d3fd2946ecc4634
[INFO 24-02-11 19:18:38.0748 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:38.2554 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:38.2554 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:38.2554 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:60


[INFO 24-02-11 19:18:38.4727 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp9lpxzxtr/model/ with prefix 1863a51d84694301
[INFO 24-02-11 19:18:38.4742 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:38.6581 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:38.6582 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:38.6582 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:61


[INFO 24-02-11 19:18:38.9223 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpkka0c1bu/model/ with prefix 7beba034a5694c7e
[INFO 24-02-11 19:18:38.9248 IST kernel.cc:1061] Use fast generic engine


i:62


[WARNING 24-02-11 19:18:39.3405 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:39.3406 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:39.3406 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-11 19:18:39.6081 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp8bluu_mq/model/ with prefix defe31be7e284f5a
[INFO 24-02-11 19:18:39.6116 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:39.8040 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:39.8040 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:39.8040 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:63


[INFO 24-02-11 19:18:40.0128 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpny__d4hl/model/ with prefix 480b0ffd81954c07
[INFO 24-02-11 19:18:40.0145 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:40.1983 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:40.1983 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:40.1983 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:64


[INFO 24-02-11 19:18:40.4433 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp8jtedg2z/model/ with prefix 2bfe43f33d8042eb
[INFO 24-02-11 19:18:40.4460 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:40.6206 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:40.6206 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:40.6207 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:65


[INFO 24-02-11 19:18:40.8817 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpvw44zsza/model/ with prefix 43a4e80bde4b496d
[INFO 24-02-11 19:18:40.8852 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:41.0615 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:41.0615 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:41.0615 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:66


[INFO 24-02-11 19:18:41.2800 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp8s3xscu2/model/ with prefix 68710d5f16d24ee9
[INFO 24-02-11 19:18:41.2822 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:41.4792 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:41.4792 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:41.4792 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:67


[INFO 24-02-11 19:18:41.7364 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp3e8wrhe1/model/ with prefix 8160b96b1f994f9a
[INFO 24-02-11 19:18:41.7392 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:41.9142 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:41.9142 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:41.9142 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:68


[INFO 24-02-11 19:18:42.1930 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpo1dn0xvz/model/ with prefix ad79fc9700b54919
[INFO 24-02-11 19:18:42.1968 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:42.3735 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:42.3735 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:42.3735 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:69


[INFO 24-02-11 19:18:42.6563 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpgfed6rb2/model/ with prefix 6f65b3ce24b24b61
[INFO 24-02-11 19:18:42.6599 IST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-11 19:18:42.6599 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:42.8357 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:42.8357 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:42.8357 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:70


[INFO 24-02-11 19:18:43.0597 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpnrv67ojq/model/ with prefix dff125377f3043f7
[INFO 24-02-11 19:18:43.0620 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:43.2421 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:43.2422 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:43.2422 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:71


[INFO 24-02-11 19:18:43.4536 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpxahqknk6/model/ with prefix a10bc0aa10174969
[INFO 24-02-11 19:18:43.4553 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:43.6522 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:43.6522 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:43.6522 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:72


[INFO 24-02-11 19:18:43.8931 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmptytlee0j/model/ with prefix c28ea848d32548d3
[INFO 24-02-11 19:18:43.8961 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:44.0721 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:44.0722 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:44.0722 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:73


[INFO 24-02-11 19:18:44.2734 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpe5b_qwaq/model/ with prefix f72e34362722494d
[INFO 24-02-11 19:18:44.2747 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:44.4525 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:44.4525 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:44.4525 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:74


[INFO 24-02-11 19:18:44.7972 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpaymm1w3w/model/ with prefix 4adbbede477d4915
[INFO 24-02-11 19:18:44.8023 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:44.9767 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:44.9767 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:44.9767 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:75


[INFO 24-02-11 19:18:45.1916 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpd_sbch22/model/ with prefix 77dbfed5d4094e71
[INFO 24-02-11 19:18:45.1935 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:45.3699 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:45.3700 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:45.3700 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:76


[INFO 24-02-11 19:18:45.5884 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpgrema7v2/model/ with prefix 2daf1d4c40594058
[INFO 24-02-11 19:18:45.5911 IST kernel.cc:1061] Use fast generic engine


i:77


[WARNING 24-02-11 19:18:45.8058 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:45.8058 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:45.8058 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-11 19:18:46.0619 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpnstorimj/model/ with prefix 8de9ac8b6b114f86
[INFO 24-02-11 19:18:46.0652 IST kernel.cc:1061] Use fast generic engine


i:78


[WARNING 24-02-11 19:18:46.2714 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:46.2715 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:46.2715 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-11 19:18:46.4818 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpavracsh1/model/ with prefix 5cc93648526a49d9
[INFO 24-02-11 19:18:46.4831 IST kernel.cc:1061] Use fast generic engine


i:79


[WARNING 24-02-11 19:18:46.6852 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:46.6852 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:46.6852 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-11 19:18:46.9633 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpsmih3c31/model/ with prefix c423d2bb286e4a90
[INFO 24-02-11 19:18:46.9669 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:47.1409 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:47.1410 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:47.1410 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:80


[INFO 24-02-11 19:18:47.3721 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpwkgdth1d/model/ with prefix ecda599565064744
[INFO 24-02-11 19:18:47.3742 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:47.5550 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:47.5550 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:47.5550 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:81


[INFO 24-02-11 19:18:47.8149 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpwfyb4p3d/model/ with prefix 99608c004c3944bc
[INFO 24-02-11 19:18:47.8180 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:47.9932 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:47.9932 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:47.9933 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:82


[INFO 24-02-11 19:18:48.2586 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpoxa1veoz/model/ with prefix 86bf242316fc4a7e
[INFO 24-02-11 19:18:48.2622 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:48.4550 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:48.4550 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:48.4550 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:83


[INFO 24-02-11 19:18:48.7624 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpav9i110x/model/ with prefix 99f3f7952da14e73
[INFO 24-02-11 19:18:48.7659 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:48.9470 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:48.9470 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:48.9470 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:84


[INFO 24-02-11 19:18:49.2093 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmppsy1ecxs/model/ with prefix f488c27f0e274ed2
[INFO 24-02-11 19:18:49.2126 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:49.3938 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:49.3938 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:49.3938 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:85


[INFO 24-02-11 19:18:49.6840 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp3rvv4f2n/model/ with prefix c6964ea686454c0d
[INFO 24-02-11 19:18:49.6868 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:49.8607 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:49.8607 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:49.8607 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:86


[INFO 24-02-11 19:18:50.1701 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpj64zi940/model/ with prefix 131a35fcb1cd47cb
[INFO 24-02-11 19:18:50.1752 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:50.3514 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:50.3514 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:50.3514 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:87


[INFO 24-02-11 19:18:50.6048 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpw0nc5vb6/model/ with prefix c7e5536320014956
[INFO 24-02-11 19:18:50.6074 IST kernel.cc:1061] Use fast generic engine


i:88


[WARNING 24-02-11 19:18:51.0644 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:51.0645 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:51.0645 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-11 19:18:51.2772 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpbilovxzp/model/ with prefix e29173b8e4264868
[INFO 24-02-11 19:18:51.2787 IST kernel.cc:1061] Use fast generic engine


i:89


[WARNING 24-02-11 19:18:51.4919 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:51.4919 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:51.4919 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-11 19:18:51.7330 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp__bus5pa/model/ with prefix 57f7f5012a7743de
[INFO 24-02-11 19:18:51.7356 IST kernel.cc:1061] Use fast generic engine


i:90


[WARNING 24-02-11 19:18:51.9443 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:51.9444 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:51.9444 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-11 19:18:52.1772 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpd7h6bxnk/model/ with prefix a0f41dbb934a4199
[INFO 24-02-11 19:18:52.1797 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:52.3542 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:52.3542 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:52.3542 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:91


[INFO 24-02-11 19:18:52.6267 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmphimnpwqu/model/ with prefix 99f5f79939a64230
[INFO 24-02-11 19:18:52.6304 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:52.8050 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:52.8051 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:52.8051 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:92


[INFO 24-02-11 19:18:53.1942 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp8yvn2md9/model/ with prefix 7beb06e7ebce4a3f
[INFO 24-02-11 19:18:53.2007 IST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-11 19:18:53.2007 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:53.3739 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:53.3739 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:53.3739 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:93


[INFO 24-02-11 19:18:53.5856 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpzbs0l0cc/model/ with prefix 7feb425d07ae4950
[INFO 24-02-11 19:18:53.5875 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:53.7602 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:53.7602 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:53.7602 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:94


[INFO 24-02-11 19:18:53.9830 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpren998km/model/ with prefix 4bf3999f1ca041cc
[INFO 24-02-11 19:18:53.9854 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:54.1588 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:54.1588 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:54.1588 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:95


[INFO 24-02-11 19:18:54.3946 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpvkaa6uip/model/ with prefix c08eea73309e43ee
[INFO 24-02-11 19:18:54.3973 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:54.5691 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:54.5691 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:54.5691 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:96


[INFO 24-02-11 19:18:54.8434 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmph_7698op/model/ with prefix 14c45ad981fb43d7
[INFO 24-02-11 19:18:54.8462 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:55.0251 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:55.0251 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:55.0251 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:97


[INFO 24-02-11 19:18:55.2373 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpdc932yq_/model/ with prefix b2a2b3e2911f4f95
[INFO 24-02-11 19:18:55.2392 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:55.4149 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:55.4149 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:55.4149 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:98


[INFO 24-02-11 19:18:55.7805 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp9cjcdre0/model/ with prefix 208119eef0574764
[INFO 24-02-11 19:18:55.7877 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-11 19:18:55.9638 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:55.9638 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-11 19:18:55.9638 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:99
Submission exported to submission.csv


[INFO 24-02-11 19:18:56.4127 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpwxiuyss9/model/ with prefix 934b5c07b4104931
[INFO 24-02-11 19:18:56.4224 IST kernel.cc:1061] Use fast generic engine


# What is next

If you want to learn more about TensorFlow Decision Forests and its advanced features, you can follow the official documentation [here](https://www.tensorflow.org/decision_forests) 